### PR TITLE
Milestone - Docs

### PR DESCRIPTION
**Pull request by newhorizon-tech**
- Brainstormed ideas on how to implement "Invite Friend" feature
- Created an Entity Relationship Diagram for our data model

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've done well in this milestone.
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Your GitHub Actions checks are not running.
- Although you did not do any coding in this milestone. Yet, you should ensure that the GitHub Actions checks are running and all checks are successful. This is a requirement for all milestones in this project.
- [x] The cardinalities of the relationships are not well represented.
- I observed that you used `1` for one and `*` for many. This is correct but can be wrongly represented if care is not taken. you should use the suggested way of representing the relationships. You should use [the recommended online tool](https://www.lucidchart.com/pages/how-to-draw-ERD) to draw your ERD.
- [x] Your entity (table) names should be represented in the singular form, using title case. 
- For instance, you should have `User` and not `users` etc.
- [x] You have to clearly and correctly represent your primary and foreign keys
- I assume that you indicated your primary keys by bolding the `id's` for each entity. However, this can be misinterpreted.
- Primary keys should be clearly represented with `PK`
- Foreign keys should be clearly represented with `FK`



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by elmejdki — COMMENTED** [review]
# status: approved :heavy_check_mark: 

Great job @team, thanks for making the required changes.

now you can merge, your project is approved.

## Happy Coding :tophat:

**Pull request merged**
